### ISSUE TITLE
please add stock and option data schema and definitions

### ISSUE THREAD

**Issue by bionicles** [westonplatter/fast_arrow#68]
**What's your feature?**
it's hard to know what the data mean

**Why are you interested in it being added?**
```
>>> print(spy.keys())
dict_keys(['margin_initial_ratio', 'rhs_tradability', 'id', 'market', 'simple_name', 'min_tick_size', 'maintenance_ratio', 'tradability', 'state', 'type', 'tradeable', 'fundamentals', 'quote', 'symbol', 'day_trade_ratio', 'name', 'tradable_chain_id', 'splits', 'url', 'country', 'bloomberg_unique', 'list_date'])
```
```
>>> print(options[0].keys())
dict_keys(['issue_date', 'tradability', 'strike_price', 'state', 'url', 'expiration_date', 'created_at', 'chain_id', 'updated_at', 'rhs_tradability', 'type', 'chain_symbol', 'min_ticks', 'id'])
```

**Comment by bionicles** [comment]
how is tradability different from rhs_tradability?

**Comment by westonplatter** [comment]
hey @bionicles! Thanks for the suggestion. I don't have time to take this on, but totally willing to add it as a feature if there's a Pull Request.

In response to your question, 

> how is tradability different from rhs_tradability?

I don't know what the difference is. I don't have any insight into field definitions beyond intuition. As context, (1) Robinhood doesn't officially their API and therefore doesn't offer documentation on the fields. (2) the unofficial docs don't define fields for the option API endpoints (https://github.com/sanko/Robinhood/blob/master/Options.md).

**Comment by bionicles** [comment]
do the option prices only show up during active trading hours?

**Comment by westonplatter** [comment]
no, RH provides quotes bid/ask 24/7. The spread increases when the market is closed, but the API still returns data. 

If you want to see this in action, place a `import ipdb; ipdb.set_trace()` on line 64 of `option_order_place_single.py`.
